Tools/build/generate_sbom.py

```diff
@@ -161,6 +163,23 @@ def get_externals() -> list[str]:
     return externals
 
 
+def download_with_retries(
+    download_location: str,
+    retries: int = 5) -> typing.Any:
```

Comment:
I think we have a more general way to move arguments to a new line:
```suggestion
def download_with_retries(download_location: str,
                          retries: int = 5) -> typing.Any:
```
but I think that `Any` can be gotten rid of:
```suggestion
def download_with_retries(download_location: str, retries: int = 5):
```